### ISSUE TITLE
Values of Learning rate, scheduler values for Ours-HXE

### ISSUE THREAD

**Issue by 07Agarg** [sindish/unitedwelearn#1]
Hi @sindish, Thank you for the repository!
```
parser.add_argument("--weights", type=float, nargs="+", default=None, help="Weights for hierarchical cross entropy.")
parser.add_argument("--multistep_lr", type=boolean, default=False, help="use multi step learning rate")
parser.add_argument("--lr_steps", type=int, nargs="+", default=[35], help="Steps for learning rate decrease.")
```
I am trying to replicate the results of your paper. You have used the above additional arguments for training with your proposed loss function but I didn't find the default values of the same. Can you provide the exact python command for running with your loss function in both settings - (a) Ours-HXE (leaf-focused weighting) and (b) Ours-HXE (hier-focused weighting)? The original repository of Making Better Mistakes has included bash files (with python commands) for each loss function. 

Thanks!
